package.json

```diff
@@ -39,8 +39,8 @@
     "build": "REACT_APP_DEPLOY_PRIME_URL=\"$DEPLOY_PRIME_URL\" react-scripts build",
     "test": "react-scripts test",
     "coverage": "react-scripts test --coverage --watchAll=false",
-    "lint": "eslint src && prettier --check src/locales/en.json *.json",
-    "fix": "eslint src --fix && prettier --write src/locales/en.json *.json",
+    "lint": "black *.py --diff && eslint src && prettier --check src/locales/en.json *.json",
```

Comment:
Have you checked that this will fail CI if the format isn't correct?

Reply:
I've checked that it will fail locally but not in CI